scripts/prune-environment.py

```diff
@@ -1,3 +1,5 @@
+from typing import Iterable, List
```

Comment:
how to spot a PR from @vringar

Reply:
Sorry :grin: 
But I got https://github.com/mozilla/OpenWPM/blob/master/CONTRIBUTING.md#types-annotations-in-python into the repo I feel like I have a justification. ^^
It also helped me understand the code.